projectile.el

```diff
@@ -1587,12 +1609,15 @@ If `command' is nil or an empty string, return nil.
 This allows commands to be disabled.
 
 Only text sent to standard output is taken into account."
-  (when (stringp command)
+  (when (or (stringp command)
+             (functionp command))
     (let ((default-directory root))
+      (if (functionp command)
```

Comment:
I'm puzzled by this code, as you call the command, but don't do anything with the result.